app/controllers/users_controller.rb

```diff
@@ -57,25 +63,12 @@ def set_following_relationship
   end
 
   def charge_user_with_stripe
-    
-    # Set your secret key: remember to change this to your live secret key in production
-    # See your keys here https://dashboard.stripe.com/account/apikeys
-    Stripe.api_key = ENV['STRIPE_SECRET_KEY']
-
-    # Get the credit card details submitted by the form
+    StripeWrapper.set_api_key
```

Comment:
This should be inside wrapper if so. Or set the api key in initializer :) but you will learn more in next assignments.